src/builders.ts

```diff
@@ -18,7 +18,7 @@ import TodoMatcher from './todo-matcher';
 const SEPARATOR = '|';
 
 export function buildFromTodoOperations(
-  todoOperations: string[],
+  todoOperations: Operation[],
```

Comment:
now that the operations array isn't polluted with empty string, we can use the correct `Operation[]` type here